Kitodo/src/test/java/org/kitodo/production/services/data/ProjectServiceTest.java

```diff
@@ -52,11 +49,6 @@ public void testProjectForCompletness() throws IOException {
         project.setTitle("testProject");
         Assert.assertFalse("Project shouldn't be complete", projectService.isProjectComplete(project));
 
-        // Add templates, still not complete
```

Comment:
why did you remove this?

Reply:
Since the F field is removed, the test had to be adapted.